Tests/Configuration/appConfig.json

```
@@ -2,5 +2,7 @@
     "url": "",
     "rnwAdminUrl": "",
     "username":"",
-    "password":""
+    "password":"",
```

Can refer password as B2C user.
```suggestion
    "B2CAutoTest_Pass":"",
```

---

Variable name changed